test/test_jenkins_set_options.py

```diff
@@ -144,3 +144,9 @@ def testDelRoot(self):
         self.executeBobJenkinsCmd("push -q myTestJenkins")
         send = self.jenkinsMock.getServerData()
         assert(send[0][0] == '/job/test/doDelete')
+
+    def testShortDescription(self):
```

Comment:
Minor: Maybe you want to extend the testcase (add a more complex recipe) and parse the uploaded description with `--shortdescription` and `--longdescription`?